.gitignore

```diff
@@ -23,3 +23,4 @@ cmd/errorutil/errorutil
 **errorutil_errors_export.json
 *.DS_Store
 coverage.txt
+.idea
```

Comment:
Why this??

Reply:
it is like `.vscode/`  for jetbrains ide